pkg/ddevapp/wordpress.go

```diff
@@ -100,75 +137,73 @@ define('DB_PASSWORD', '{{ $config.DatabasePassword }}');
 /** MySQL hostname */
 define('DB_HOST', '{{ $config.DatabaseHost }}');
 
-/** Database Charset to use in creating database tables. */
-define('DB_CHARSET', 'utf8mb4');
+/** Enable debug */
+define('WP_DEBUG', true);
 
-/** The Database Collate type. Don't change this if in doubt. */
-define('DB_COLLATE', '');
+/** WP_HOME URL */
+define('WP_HOME', '{{ $config.DeployURL }}');
 
-/**
- * WordPress Database Table prefix.
- */
-$table_prefix  = '{{ $config.TablePrefix }}';
-
-/**
- * For developers: WordPress debugging mode.
- */
-define('WP_DEBUG', false);
+/** WP_SITEURL location */
+define('WP_SITEURL', WP_HOME . '/{{ $config.AbsPath  }}');
```

Comment:
I think @alkymst said this wasn't necessary, but I don't think it does harm?

Reply:
They are both necessary if WP is in a sub-directory. It's more failsafe in general too.